deepspeed/runtime/zero/stage_1_and_2.py

```diff
@@ -1691,7 +1695,7 @@ def step(self, closure=None):
         self.start_timers([OPTIMIZER_GRADIENTS])
         norm_groups = []
         single_partition_grad_groups = []
-        skip = False
+        # skip = False
```

Comment:
It is okay to remove `skip` if not used.

Reply:
> It is okay to remove `skip` if not used.

It's not used, just comment it for safe.